<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>660</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="text">
    <string>摄像头识别</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../yolov5-7.0/UI设计/images/摄像头_camera-one.png</normaloff>../../yolov5-7.0/UI设计/images/摄像头_camera-one.png</iconset>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>580</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>图片识别</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../yolov5-7.0/UI设计/images/图片收集_collect-picture.png</normaloff>../../yolov5-7.0/UI设计/images/图片收集_collect-picture.png</iconset>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>740</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.0227727, y1:0.472, x2:0.977273, y2:0.476955, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
	font: 75 16pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-radius: 20
}
QPushButton:hover{
background-color:qlineargradient(spread:pad, x1:0.198909, y1:0.302, x2:0.892046, y2:0.687, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
padding-bottom: 7;

}
QPushButton:pressed{
padding-top: 7;
padding-left: 7;
}
QPushButton:disabled{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(119, 119, 119, 255), stop:1 rgba(221, 221, 221, 255));
	color: rgba(241, 241, 241, 150);
}</string>
   </property>
   <property name="text">
    <string>调用detect</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>740</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.0227727, y1:0.472, x2:0.977273, y2:0.476955, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
	font: 75 16pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-radius: 20
}
QPushButton:hover{
background-color:qlineargradient(spread:pad, x1:0.198909, y1:0.302, x2:0.892046, y2:0.687, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
padding-bottom: 7;

}
QPushButton:pressed{
padding-top: 7;
padding-left: 7;
}
QPushButton:disabled{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(119, 119, 119, 255), stop:1 rgba(221, 221, 221, 255));
	color: rgba(241, 241, 241, 150);
}</string>
   </property>
   <property name="text">
    <string>开始识别</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>0</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
padding-bottom: 7;
padding-left: 7;
background-color: rgba(255, 255, 255, 0);
}
QPushButton{
background-color: rgba(255, 255, 255, 0);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/关闭_close.png</normaloff>images/关闭_close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>781</width>
     <height>891</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30;
background-color: rgba(255, 255, 255, 150);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/bg.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
padding-bottom: 7;
background-color: rgba(255, 255, 255, 0);
}
QPushButton{
background-color: rgba(255, 255, 255, 0);
	color: rgb(0, 170, 255);
	font: 12pt &quot;微软雅黑&quot;;
}
</string>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/帮助_help.png</normaloff>images/帮助_help.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
padding-bottom: 7;
background-color: rgba(255, 255, 255, 0);
}
QPushButton{
background-color: rgba(255, 255, 255, 0);
	color: rgb(0, 170, 255);
	font: 12pt &quot;微软雅黑&quot;;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/logo.png</normaloff>images/logo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>comboBox</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>704</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
